tests/test_yaml.py

```diff
@@ -16,7 +16,8 @@
 )
 def test_yaml_update_from_source(tmpdir, original_yaml, source_python, expected_yaml):
     yaml_path = os.path.join(tmpdir, "valohai.yaml")
-    source_path = os.path.join(tmpdir, "test.py")
+    filename, file_extension = os.path.splitext(source_python)
+    source_path = os.path.join(tmpdir, f"test{file_extension}")
```

Comment:
the same mention about unused variables introduced in the code, prefix or replace with `_` in the future